_docs_integrate/iql-syntax.md

```diff
@@ -0,0 +1,80 @@
+---
+# Start automatic generation
+permalink: integrate/iql-syntax
+published: true
+title: "IQL Syntax"
+type: scenario
+toc: true
+properties:
+  - id: SC048
+  - category: Manage Attributes
+  - description: Description of the IQL Syntax
+  - customer: All
+  - component: integrate
+  - level: Advanced
+  - implementation status: DONE
+  - documentation status: DONE
+  - published: true
+  - link: iql-syntax
+require:
+required_by:
+# End automatic generation
+---
+
+The Enmeshed _Identity Query Language_ (IQL) is a domain-specific language to query IdentityAttributes using a concise and simple syntax. The IQL is tailored towards usage by non-technical users and integrators. The IQL is complete, i.e. it's expressive enough to query arbitrary subsets of IdentityAttributes and can thus serve as a replacement of the other types of queries dealing with IdentityAttributes.
+
+## Basic Syntax
+
+### Filter by Value Type
+
+The most common usage of the IQL is to filter IdentityAttributes by their value types or tags. Value types are specified by directly using their names as terms in the query string. E.g. the following IQL query will match all phone numbers, i.e. all IdentityAttributes of value type _PhoneNumber_:
+
+```iql
+PhoneNumber
+```
+
+### Filter by Tags
+
+Tags can be filtered for by prefacing the tag with a hash _#_. E.g. the following IQL query will match all IdentityAttributes whose tags contain the value _emergency_:
+
+```iql
+#emergency
+```
+
+### Conjunctions of Terms
+
+Multiple query terms can be combined by using the and-operator `&&` or the or-operator `||` to retrieve the intersection or the union of the attributes matching the subqueries. E.g. in order to filter all the phone numbers which contain the _emergency_ tag use to following IQL query:
```

Comment:
Typo: "...use **the** following IQLQuery:" instead of "...use to following IQLQuery:"